src/NHibernate/Util/ReflectHelper.cs

```diff
@@ -104,6 +104,82 @@ public static MethodInfo GetMethod(Expression<System.Action> method)
 			return ((MethodCallExpression)method.Body).Method;
 		}
 
+		/// <summary> Get a <see cref="MethodInfo"/> from a method group </summary>
+		/// <param name="func">A method group</param>
+		internal static MethodInfo FastGetMethod<TResult>(System.Func<TResult> func)
+		{
+			return func.Method;
+		}
+
+		/// <summary> Get a <see cref="MethodInfo"/> from a method group </summary>
+		/// <param name="func">A method group</param>
+		/// <param name="a">A dummy parameter</param>
+		internal static MethodInfo FastGetMethod<T, TResult>(System.Func<T, TResult> func, T a)
+		{
+			return func.Method;
+		}
+
+		/// <summary> Get a <see cref="MethodInfo"/> from a method group </summary>
+		/// <param name="func">A method group</param>
+		/// <param name="a1">A dummy parameter</param>
+		/// <param name="a2">A dummy parameter</param>
+		internal static MethodInfo FastGetMethod<T1, T2, TResult>(System.Func<T1, T2, TResult> func, T1 a1, T2 a2)
+		{
+			return func.Method;
+		}
+
+		/// <summary> Get a <see cref="MethodInfo"/> from a method group </summary>
+        /// <param name="func">A method group</param>
+		/// <param name="a1">A dummy parameter</param>
+		/// <param name="a2">A dummy parameter</param>
+		/// <param name="a3">A dummy parameter</param>
+        internal static MethodInfo FastGetMethod<T1, T2, T3, TResult>(System.Func<T1, T2, T3, TResult> func, T1 a1, T2 a2, T3 a3)
```

Comment:
Indentation glitches here and a few lines above.

Reply:
Fixed, thanks.